service/witness/witness/witness.go

```
@@ -290,3 +293,11 @@ func (w *Witness) constructBlockWitness(block *block.Block, latestVerifiedBlockN
 	}
 	return &blockWitness, nil
 }
+
+func (w *Witness) Shutdown() {
```

Can we return the errors and log it if `err !=nil`

---

changed the codes to log errors